src/common/util.js

```diff
@@ -401,7 +401,7 @@ var Cookies = {
       return loginids.map(function(_id) { return _id.id }).indexOf(id.id) === -1;
     });
 
-    return oauth_loginids ? oauth_loginids : loginids
+    return oauth_loginids.length > 0 ? oauth_loginids : loginids
```

Comment:
@4p00rv Do this
`oauth_loginids && oauth_loginids.length > 0 ? oauth_loginids : loginids`